lib/big_query/timestamp_rollup.ex

```diff
@@ -4,6 +4,7 @@ defmodule BigQuery.TimestampRollup do
   @callback is_a?(String.t) :: boolean
   @callback floor(%DateTime{}) :: %DateTime{}
   @callback ceiling(%DateTime{}) :: %DateTime{}
-  @callback range(%DateTime{}, %DateTime{}, boolean) :: [%DateTime{}]
+  @callback next(%DateTime{}) :: %DateTime{}
```

Comment:
Added another helper method `next()`.  I found myself often wanting to get the next timestamp for the interval ... which is not the same as ceiling().